packages/backend/src/apps/telegram-bot/common/throw-errors.ts

```diff
@@ -35,6 +40,25 @@ export function throwSendMessageError(
           err,
         )
       } else if (errorString.includes('supergroup')) {
+        // SPECIAL CASE: fix chat id for user if pipe is published
+        if (!$.execution.testRun) {
+          const oldChatId: string = $.step.parameters.chatId as string
+          const newChatId: string =
+            err.response.data.parameters['migrate_to_chat_id']
```

Comment:
can we check that `newChatId` is defined before updating. we cant possible be sure that this will always be returned in the error. furthermore, the check for this error - includes('supergroup') is a bit lax